chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/RoomService.kt

```diff
@@ -78,33 +88,33 @@ internal class RoomService(
                     customData = customData,
                     userIds = userIds
             ).toJson()
-                    .flatMap { body -> client.doPost<Room>("/rooms", body) }
-                    .map { room ->
-                        roomStore += room
-                        userService.populateUserStore(room.memberUserIds)
-                        room
+                    .flatMap { body -> client.doPost<CreateRoomResponse>("/rooms", body) }
+                    .map { response ->
+                        roomStore.add(response)
+                        userService.populateUserStore(response.membership.userIds.toSet())
+                        mapToRoom(response.room, response.membership, null)
```

Comment:
When creating a room shouldn't unread count be known and equal to 0?

Reply:
You could infer that from the fact that the room has just been created, yes, but it's not part of the backend response.